AGENTS.md

```
@@ -31,103 +31,52 @@ yamllint --strict .
 
 ## Architecture
 
-### Kustomize Composition
+### Helm Chart
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)